change/@fluentui-react-table-f1b9fee8-bec3-49cb-b249-88f7dbfdcd5b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "#28562: [Bug]: DataGrid Headers look weird on Safari",
```

Comment:
```suggestion
  "comment": "fix: DataGrid Headers look weird on Safari",
```

Reply:
Hi @sopranopillow, thanks for the correction. I've updated the comment and pushed changes.